src/amchart/CommonXY.js

```diff
@@ -124,10 +124,11 @@
         if (this.xAxisType() === "ordinal") {
             this._chart.valueAxes[0].integersOnly = true;
             this._chart.valueAxes[0].labelFunction = function(val) {
-                if (val > 0) {
-                    return typeof(context.data()[val]) !== "undefined" ? context.data()[val][0] : ""; // shift chart right by 1 so it does not start on axis
-                }
-                return "";
+                return context.data()[--val][0];
+                // if (val > 0) {
```

Comment:
leaving this commented code in there on purpose (i think amcharts changed something so i have to do this diff)